cmd/cluster-init/testdata/ci-operator/jobs/infra-periodics.yaml

```diff
@@ -0,0 +1,45 @@
+periodics:
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+    - base_ref: master
+      org: openshift
+      repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-openshift-release-master-existingCluster-apply
+  spec:
+    containers:
+      - args:
+          - --config-dir=clusters/build-clusters/exisingCluster
+          - --confirm=true
+          - --as=
```

Comment:
uh oh

Reply:
Looks like this is the way it is in the `release/ci-operator/jobs/infra-periodics.yaml` file...am I missing something?

Reply:
:exploding_head:

Reply:
@hongkailiu why?